### PR TITLE
Bowl escape decoder transfer training and improving latent regularization

### PR DESCRIPTION
**Pull request by charles-zhng**
- split latent space regularization into two terms: kl reg (KL divergence from standard Gaussian), and AR1 reg (L2 from previous latent), with their own respective weight configs and schedules
- implement vf_coefficient parameter (scales value function loss). previously was hardcoded to 0.5.
- implement cleaner decoder reloading logic for the decoder inference function used in high-level task policy training.
- update rendering to use `env._suffix` for getting camera name
- add gradient clipping metrics logging (cred: @scott-yj-yang)
- remove unused files
- update default rodent config accordingly (TODO: update/include other animal configs)
- cleanup checkpointing.py
- remove unused config values, add vnl-playground commit hash to config

### CODE REVIEW

File: track_mjx/agent/mlp_ppo/ppo.py [hunk 1]
```diff
@@ -572,15 +575,22 @@ def reset_fn_donated_env_state(env_state_donated, key_envs):
     make_logging_policy = ppo_networks.make_logging_inference_fn(ppo_network)
     jit_logging_inference_fn = jax.jit(make_logging_policy(deterministic=True))
 
+    grad_clip_threshold = 20.0
```
Comment (asirbu77) [inline track_mjx/agent/mlp_ppo/ppo.py]: Maybe put this as a config parameter instead
